core_lib/interface/editor.cpp

```diff
@@ -718,6 +722,37 @@ bool Editor::exportSeqCLI(QString filePath, QString format, int width, int heigh
     return true;
 }
 
+bool Editor::exportMovieCLI(QString filePath, int width, int height)
```

Comment:
Exporting a movie through the GUI allows for range selection or running to the end of the sound layer. Exporting both image sequences and movies allows for camera layer selection. There should be optional arguments for all of these things. If we choose to use one single argument for both movie and image sequence export, then we should probably throw warnings for any options that will be ignored.

Reply:
Yeah, good point here too. I just blindly followed what the image sequence exporter was doing without thinking too much about it, but these are indeed reasonable enhancements. I’ll take care of it asap.